sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/protocol/blob_rest_client.hpp

```diff
@@ -728,10 +634,44 @@ namespace Azure { namespace Storage { namespace Blobs {
       std::string ETag;
       std::string LastModified;
       std::string CopyId;
-      Blobs::Models::CopyStatus CopyStatus = Blobs::Models::CopyStatus::Unknown;
+      Models::CopyStatus CopyStatus = Models::CopyStatus::Unknown;
       Azure::Core::Nullable<std::string> VersionId;
     }; // struct StartCopyPageBlobIncrementalResult
 
+    struct AppendBlockFromUriResult
```

Comment:
Why the sequence of the model types are changed?

Reply:
They're not changed, they're reordered. This is a defect of this auto-generator.